### PR TITLE
Add kubenertes support for qlora finetune

### PR DESCRIPTION
**Pull request by Romanticoseu**
## Description

Add kubenertes support for qlora finetune

### CODE REVIEW

**Comment by Romanticoseu** [comment]
Need to test

**Comment by Romanticoseu** [comment]
In standalone model
image

**Comment by Uxito-Ada** [comment]
pls remove all the files that have not been tested

**Comment by Uxito-Ada** [comment]
> In standalone model image

Why the standalone is in k8s PR? they are two things

File: docker/llm/finetune/lora/cpu/docker/Dockerfile [hunk 1]
```diff
@@ -19,7 +19,7 @@ COPY --from=key-getter /root/intel-oneapi-archive-keyring.gpg /usr/share/keyring
 RUN echo "deb [signed-by=/usr/share/keyrings/intel-oneapi-archive-keyring.gpg] https://apt.repos.intel.com/oneapi all main " > /etc/apt/sources.list.d/oneAPI.list
 
 RUN mkdir /bigdl/data && mkdir /bigdl/model && \
-# install pytorch 2.0.1
+# install pytorch 2.1.0
```
Comment (Uxito-Ada) [inline docker/llm/finetune/lora/cpu/docker/Dockerfile:22]: why 2.1.0? the below ipex version is still 2.0.1, which is inconsistent to torch 2.1.0

Comment (Romanticoseu) [reply docker/llm/finetune/lora/cpu/docker/Dockerfile:22]: I only edited comments, the torch version has been modified in this [commit](https://github.com/intel-analytics/BigDL/commit/30d14c51a62ea92db886de188ff63dd9ef6b8c91)

**Pull request closed without merging**